tools/db_stress.cc

```diff
@@ -1713,6 +1713,7 @@ class StressTest {
     }
     ReadOptions ropt;
     ropt.snapshot = snap_state.snapshot;
+    ropt.total_order_seek = true;
```

Comment:
Could you remind me why `total_order_seek` needs to be true here?

Reply:
When `-memtablerep=prefixhash` (the default for db_stress) and `total_order_seek=false`, `SeekToFirst`, which is used on L1741 below, is unsupported: https://github.com/facebook/rocksdb/blob/8c2a48742a00670fb05752fb34d72ba7f2ba3150/memtable/hash_skiplist_rep.cc#L193-L195.

There may be other places lacking support because generally I think we only guarantee seeking/scanning across prefixes works when `total_order_seek=true`.